unison-src/transcripts/pattern-pretty-print-2345.output.md

```diff
@@ -183,8 +183,8 @@ doc = cases
 
   throaty : Request {g, Ab} x -> ()
   throaty = cases
-    {Ab.a a -> k} -> ()
-    { _ }         -> ()
+    { Ab.a a -> k } -> ()
+    { _ }           -> ()
```

Comment:
For some strange reason, the pure case of request matching was done with spaces after the braces, but the operations left out the spaces. WTH?

Now both use a space after the braces. I find it a bit more readable to give the symbol some breathing room.